dashboard/app/reporting_test.go

```diff
@@ -721,3 +721,62 @@ func TestAltTitles7(t *testing.T) {
 	c.expectEQ(rep.Title, crash1.Title)
 	c.expectEQ(rep.Log, crash2.Log)
 }
+
+func TestDetachExternalTracker(t *testing.T) {
+	c := NewCtx(t)
+	defer c.Close()
+
+	build := testBuild(1)
+	c.client.UploadBuild(build)
+
+	crash1 := &dashapi.Crash{
```

Comment:
There is a number of helper functions that create simple test objects if you don't care about particular contexts. Here use:
```
	crash1 := testCrash(build, 1)
```
I guess you looked at some other test, and I did not modernize all of them to use these helper functions.